util/include/crypto/eddsa/EdDSA.hpp

```diff
@@ -12,10 +12,12 @@
 //
 #pragma once
 #include "SerializableByteArray.hpp"
+#include "crypto_utils.hpp"
```

Comment:
since it's an openssl eddsa implementation, please move all three `EdDSA*.hpp` files to `crypto/openssl`

Reply:
Changed in https://github.com/arc-vmware/concord-bft/pull/12/commits/7b00c7fa490e8abfda7a7da61799a0b6625826df